firebase-perf/src/main/java/com/google/firebase/perf/transport/TransportManager.java

```diff
@@ -190,6 +194,7 @@ public void initialize(
       @NonNull Provider<TransportFactory> flgTransportFactoryProvider) {
 
     this.firebaseApp = firebaseApp;
+    projectId = firebaseApp.getOptions().getProjectId();
```

Comment:
this.projectId?

Reply:
The purpose of `this` is to differentiate the input variable and instance variable if they have the same name. In my case, since there is no input param called `projectId`, it would be fine to use `projectId` without `this`. There is no universal standard to decide if `this` should be used in this case. I followed the convention in [other places in the codebase](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-perf/src/main/java/com/google/firebase/perf/transport/TransportManager.java#L203-L206) where `this` is not used for instance variables.

Reply:
My recommendation would be to be locally consistent. I.e. if a specific file does it, I'd recommend the whole file to do it. If we want to stop the practice, then we should update the whole file.